pcdsdevices/epics_motor.py

```diff
@@ -221,6 +221,48 @@ def check_limit_switches(self):
         else:
             return "Low [] High []"
 
+    def get_low_lim(self):
+        """Get the low limit."""
+        return self.low_limit_travel.get()
+
+    def set_low_lim(self, value):
+        """
+        Set the low limit.
+
+        Parameters
+        ----------
+        value : float
+            Limit of travel in the negative direction.
+        """
+        if not self.moving:
+            # update EPICS
+            if value <= self.position:
+                self.low_limit_travel.put(value)
+            else:
+                logger.debug('Could not set motor low limit to %f at '
```

Comment:
ophyd devices typically use `self.log` for this sort of thing- I think we still have a bunch of dangling `logger` references elsewhere in the code in pcdsdevices

Reply:
ohh thank you, i'll change this

Reply:
fixed this in 3290db5